remoteit/desktop
# DESK-930 Announcements

### Description


Provide in app messaging

### Test plan



1. Sign in
2. See announcements indicator
3. Click it and see announcements
4. You should see the indicator disappear after reading.

### Ticket



<https://remoteit.atlassian.net/browse/DESK-930>
